README.md

```diff
@@ -1,90 +1,133 @@
+Run `git config core.autocrlf false` to make sure this repository does not try to convert line endings for your Windows machine.
 
 ## Installing Lando
 
 1. Follow the steps at https://docs.devwithlando.io/installation/installing.html
 
 ## Using Lando
 
-The .lando.yml file included in this repo will set you up to connect to the correct Pantheon dev environment for Portland Oregon Alpha. To initialize your local site for the first time, follow the steps below:
+The .lando.yml file included in this repo will set you up to connect to the correct Pantheon dev environment for the Portland Oregon website. To initialize your local site for the first time, follow the steps below:
 
-1. Run `lando start` to initialize your containers.
+1. From within your project root directory, run `lando start` to initialize your containers.
 2. Log in to Pantheon and generate a machine token from My Dashboard > Account > Machine Tokens.
 3. Run `lando terminus auth:login --machine-token=[YOUR MACHINE TOKEN]`, this logs your Lando instance into our Pantheon account.
 4. To make sure you don't hit rate limits with composer, log into Github and generate a personal access token and add it to your lando instance by using `lando composer config --global --auth github-oauth.github.com "$COMPOSER_TOKEN"`. (You should replace $COMPOSER_TOKEN with your generated token.) There is a handy tutorial for this at https://coderwall.com/p/kz4egw/composer-install-github-rate-limiting-work-around
-5. Run `lando pull` to get the DB and files from pantheon. This process takes a while. #grabsomecoffee
-6. You should now be able to visit https://portlandor.lndo.site in your browser.
-7. When you are done with your development for the day, run `Lando stop` to shut off your development containers.
+6. Add a settings.local.php
+    - Ask a teammate for a copy of a sane file to put at `/web/sites/default/settings.local.php`. Depending on whether you do a lot of theming you may want that file to turn on twig debugging and turn off several layers of caching.
+5. You have two options to get your database and files set up:
+    1. Lando quick start:
+        1. Run `lando pull` to get the DB and files from pantheon. This process takes a while. #grabsomecoffee
+    1. Manually import the database and files
+        1. Download the latest database and files backups from the Dev environment on Pantheon.
+        2. Move your database export into a folder in your project root called `/artifacts`. (We added this to our .gitignore, so the directory won't be there until you create it.)
+        3. From the project root, run `lando db-import portlandor_dev_2018-04-12T00-00-00_UTC_database.sql.gz`. (This is just an example, you'll need to use the actual filename of the database dump you downloaded.) 
+        4. Move your files backup into `web/sites/default/files`
```

Comment:
What is your "files backup"? How is this backup created and where is it located?

Reply:
An extra "the" exists in "From the within the artifacts directory," But I'll leave it up to you whether or not to fix this minor grammatical error.